gateway/gateway.yaml

```diff
@@ -48,30 +48,30 @@ georchestra:
         target: ${georchestra.gateway.services.console.target}
         access-rules:
         - intercept-url:
-          - /console/public/.*
-          - /console/manager/public/.*
+          - /console/public/**
+          - /console/manager/public/**
           #/console/account resources are private except account/new and account/passwordRecovery
           - /console/account/new
           - /console/account/newPassword
           - /console/account/passwordRecovery
-          - /console/account/js/.*
-          - /console/account/css/.*
-          - /console/account/fonts/.*
+          - /console/account/js/**
+          - /console/account/css/**
+          - /console/account/fonts/**
           - /console/testPage
           anonymous: true
         - intercept-url:
-          - /console/private/.*
-          - /console/manager/.*
-          - /console/.*/emails
-          - /console/.*/sendEmail # /console/sendEmail features are reserved to SUPERUSER & delegated admins
-          - /console/.*/emailTemplates
+          - /console/private/**
+          - /console/manager/**
+          - /console/*/emails
```

Comment:
missing `**` here and there

Reply:
I did not get @fvanderbiest's comments: It is not possible to have double wildcard if not at the end of the entry, else spring won't start (can't recall exactly the reason).

Reply:
Indeed. 
I was fooled by your commits on the MEL datadir (did not see the last one)